pkg/cmd/cli/cmd/importer/dockercompose.go

```diff
@@ -0,0 +1,182 @@
+package importer
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/apimachinery/registered"
+	"k8s.io/kubernetes/pkg/kubectl"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/runtime"
+
+	"github.com/openshift/origin/pkg/client"
+	cmdutil "github.com/openshift/origin/pkg/cmd/util"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	configcmd "github.com/openshift/origin/pkg/config/cmd"
+	"github.com/openshift/origin/pkg/generate/app"
+	appcmd "github.com/openshift/origin/pkg/generate/app/cmd"
+	"github.com/openshift/origin/pkg/generate/dockercompose"
+)
+
+const (
+	dockerComposeLong = `
+Import a Docker Compose file as OpenShift objects
```

Comment:
So here's how I see this:

1. new-app has to shed function
2. when users think "hey, I want to bring this thing I have from elsewhere and run it on OpenShift, I'm **importing** it to OpenShift"
3. Users really don't care about templates at first
4. Advanced users would love to be able to get a template from a compose, so they can tweak it and reuse it
5. Forcing a user to learn two commands for doing 3 and 4 is silly, and forcing everyone to do "foo bar | oc create -f -" is abusive.
6. We will have lots of other "import" behaviors.
7. Import != process - process is like `create`, a primitive, while this is a higher level tool